custom_components/tuya_local/devices/ble_smart_lock_kb150a.yaml

```diff
@@ -0,0 +1,178 @@
+name: Smart Key Lock
```

Comment:
This name should just be generic, "Fingerprint lock", or "Keypad lock" are OK, as to match other locks will need the same features, but will not necessarily be key safes.